libs/application/templates/financial-aid/src/forms/Application.ts

```diff
@@ -48,5 +49,24 @@ export const Application: Form = buildForm({
         }),
       ],
     }),
+    buildSection({
+      condition: (_, externalData) =>
+        ((externalData as unknown) as ExternalData).nationalRegistry?.data
+          ?.spouse !== undefined,
+      id: 'personalInterest',
+      title: m.section.personalInterest,
```

Comment:
Relationshipstatus rather than personal interest? Just wondering, feel free to ignore

Reply:
Personal interest is for a lack of better translation 'Persónuhagir' which is this section of the form, if you have a better translation, please share 😬